frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/EmbedCodePane.tsx

```diff
@@ -0,0 +1,175 @@
+    return null;
+  }
+
+  if (variant === "parameters") {
+    const hasCodeDiff =
+      !_.isEqual(initialPreviewParameters, params) &&
+      selectedServerCodeOption.parametersSource !==
+        getEmbedServerCodeExampleOptions({
+          siteUrl,
+          secretKey,
+          resourceType,
+          resourceId: resource.id,
+          params: initialPreviewParameters,
+          displayOptions,
+        }).find(({ name }) => name === selectedServerCodeOptionName)
+          ?.parametersSource;
+
+    return (
+      <div className={className}>
+        <CodeSample
+          dataTestId="embed-backend"
+          title={t`In addition to publishing changes, update the params in the payload, like this:`}
+          selectedOptionName={selectedServerCodeOptionName}
+          languageOptions={serverCodeOptions.map(({ name }) => name)}
+          source={selectedServerCodeOption.source}
+          textHighlightMode={selectedServerCodeOption.mode}
+          highlightedText={selectedServerCodeOption.parametersSource}
+          isHighlightedTextAccent={hasCodeDiff}
+          onChangeOption={setSelectedServerCodeOptionName}
+        />
+      </div>
+    );
+  }
+
+  if (variant === "appearance") {
+    const hasCodeDiff = !_.isEqual(DEFAULT_DISPLAY_OPTIONS, displayOptions);
+    return (
+      <div className={className}>
+        <CodeSample
+          dataTestId="embed-backend"
+          title={t`Here’s the code you’ll need to alter:`}
+          selectedOptionName={selectedServerCodeOptionName}
+          languageOptions={serverCodeOptions.map(({ name }) => name)}
+          source={selectedServerCodeOption.source}
+          textHighlightMode={selectedServerCodeOption.mode}
+          highlightedText={selectedServerCodeOption.iframeUrlSource}
+          isHighlightedTextAccent={hasCodeDiff}
+          onChangeOption={setSelectedServerCodeOptionName}
+        />
+      </div>
+    );
+  }
+
+  return (
+    <Stack spacing="2rem" className={className}>
+      <CodeSample
+        dataTestId="embed-backend"
+        title={t`Insert this code snippet in your server code to generate the signed embedding URL `}
+        selectedOptionName={selectedServerCodeOptionName}
+        languageOptions={serverCodeOptions.map(({ name }) => name)}
+        source={selectedServerCodeOption.source}
+        textHighlightMode={selectedServerCodeOption.mode}
+        onChangeOption={setSelectedServerCodeOptionName}
+      />
+      <CodeSample
+        dataTestId="embed-frontend"
+        title={t`Then insert this code snippet in your HTML template or single page app.`}
+        selectedOptionName={selectedClientCodeOptionName}
+        languageOptions={clientCodeOptions.map(({ name }) => name)}
+        source={selectedClientCodeOption.source}
+        textHighlightMode={selectedClientCodeOption.mode}
+        onChangeOption={setSelectedClientCodeOptionName}
+      />
+
+      {withExamplesLink && (
+        <div className="text-centered mb2 mt2">
```

Comment:
Is it possible to use Mantine components instead of using global classes? It seems we're trying to avoid using global class as much as possible as [we've planned to migrate it.](https://www.notion.so/metabase/83-Prepare-global-css-for-migration-to-css-modules-4ab29b8589b545e0960bf032d24a11a9)